dart/dynamics/Joint.h

```diff
@@ -347,6 +354,16 @@ class Joint : public virtual common::Subject
   /// Get upper limit for position
   virtual double getPositionUpperLimit(size_t _index) const = 0;
 
+  /// Gets whether a generalized coordinate is cyclic. Returns true if and only
```

Comment:
If it depends on personal preference implicating the subject as "You" or "This function", I'd like to vote on "You" and recommend to have no `s` for consistency for now. If we all agree to have `s` with some strong reason later then let's follow the convention at that time.

Also, I'm aware of that some comments in DART begin with "Return". Let's fix them when we set the convention with documentation.